packages/plugins/Gfi/src/components/List.vue

```diff
@@ -0,0 +1,159 @@
+          )} ${Math.min((page + 1) * pageLength, listFeatures.length)} ${$t(
+            'common:plugins.gfi.list.of'
+          )} ${listFeatures.length}`
+        }}
+        <v-pagination
+          v-model="_page"
+          color="primary"
+          :length="maxPage"
+          :current-page-aria-label="
+            $t('common:plugins.gfi.list.pagination.currentPage', {
+              page: _page,
+              maxPage,
+            })
+          "
+          :page-aria-label="
+            $t('common:plugins.gfi.list.pagination.page', {
+              page: _page,
+              maxPage,
+            })
+          "
+          :next-aria-label="$t('common:plugins.gfi.list.pagination.next')"
+          :previous-aria-label="
+            $t('common:plugins.gfi.list.pagination.previous')
+          "
+          :wrapper-aria-label="$t('common:plugins.gfi.list.pagination.wrapper')"
+        ></v-pagination>
+      </v-card-subtitle>
+      <v-list>
+        <v-list-item v-if="!visibleListFeatures.length">
+          {{ $t('common:plugins.gfi.list.emptyView') }}
+        </v-list-item>
+        <v-list-item
+          v-for="feature of visibleListFeatures"
+          :key="`gfi-feature-list-${feature.ol_uid}`"
+          :two-line="listText.length === 2"
+          :three-line="listText.length === 3"
+          @click="itemClick(feature)"
+        >
+          <v-list-item-content>
+            <component
+              :is="index === 0 ? 'v-list-item-title' : 'v-list-item-subtitle'"
+              v-for="(_, index) of listText"
+              :key="`gfi-feature-list-${feature.ol_uid}-${index}`"
+            >
+              {{ $t(applyListText(feature, index)) }}
+            </component>
+          </v-list-item-content>
+        </v-list-item>
+      </v-list>
+    </v-card>
+  </v-scroll-x-reverse-transition>
+</template>
+
+<script lang="ts">
+import { Feature } from 'ol'
+import Vue from 'vue'
+import { mapGetters, mapMutations, mapActions } from 'vuex'
+
+export default Vue.extend({
+  name: 'GfiList',
+  computed: {
+    ...mapGetters('plugin/gfi', [
+      'listFeatures',
+      'listText',
+      'page',
+      'gfiConfiguration',
+    ]),
+    _page: {
+      get() {
+        return this.page + 1
+      },
+      set(value) {
+        this.setPage(value - 1)
+      },
+    },
+    pageLength() {
+      return (
+        this.gfiConfiguration.featureList.pageLength || Number.MAX_SAFE_INTEGER
+      )
+    },
+    maxPage() {
+      return Math.ceil(this.listFeatures.length / this.pageLength)
+    },
+    showPagination() {
+      return Boolean(this.gfiConfiguration.featureList.pageLength)
+    },
+    visibleListFeatures() {
+      return this.listFeatures.slice(
+        (this._page - 1) * this.pageLength,
+        this._page * this.pageLength
+      )
+    },
+  },
+  watch: {
+    maxPage: function (nextMaxPage) {
+      if (this._page > nextMaxPage) {
+        this._page = Math.max(nextMaxPage, 1)
```

Comment:
```suggestion
        this._page = nextMaxPage
```
Why not just like this?

Reply:
Pages are indexed by their numeric value, that is, they start at 1. This is due to the API provided by `v-pagination`. `nextMaxPage` might be 0 when 0 features are in the view, switching the page to 0, breaking the `v-pagination`. After getting features back into view, they will not be displayed, as nothing is on page 0.